brands/public_transport/station.json

```diff
@@ -0,0 +1,16 @@
+{
+  "public_transport/station|Greyhound Station": {
+    "countryCodes": ["us"],
+    "matchNames": ["greyhound terminal"],
+    "tags": {
+      "amenity": "bus_station",
+      "brand": "Greyhound Lines",
+      "brand:wikidata": "Q755309",
+      "brand:wikipedia": "en:Greyhound Lines",
+      "bus": "yes",
+      "name": "Greyhound Station",
+      "name:en": "Greyhound Station",
+      "public_transport": "station"
```

Comment:
You might want to add a short name of just Greyound here so we get that as a matcher. Also thanks for doing this one. I have a note with all the brands I run across that need an entry and this was in it.

Reply:
Also there's Greyhound Canada that might be worth adding while you're in this file.

Reply:
The name matcher only matches against features with the base tags, right? There could be various bars or other places called just "Greyhound".

And yes, we should add the other Greyhound subsidiaries once this is merged.